From 0af572afccbb7d1f053f49f301e9bcad02c067a3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Trung=20L=C3=AA?= <trung.le@ruby-journal.com>
Date: Wed, 5 Feb 2020 16:33:36 +1100
Subject: [PATCH] update makedepend with patched version

---
 Formula/makedepend.rb | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/Formula/makedepend.rb b/Formula/makedepend.rb
index 9cd97362b0..a5064abb88 100644
--- a/Formula/makedepend.rb
+++ b/Formula/makedepend.rb
@@ -14,10 +14,11 @@ class Makedepend < Formula
 
   depends_on "pkg-config" => :build
 
+  # The config.guess could not pick up ppc64linux
+  # Update this to version 2019
   resource "xproto" do
-    url "https://xorg.freedesktop.org/releases/individual/proto/xproto-7.0.31.tar.gz"
-    mirror "http://xorg.mirrors.pair.com/individual/proto/xproto-7.0.31.tar.gz"
-    sha256 "6d755eaae27b45c5cc75529a12855fed5de5969b367ed05003944cf901ed43c7"
+    url "https://raw.githubusercontent.com/runlevel5/linuxbrew-ppc64le/master/xproto-7.0.31-with-latest-config-guess.tar.gz"
+    sha256 "078255d5e2c2204b378729d1d47ff3f92c6dd6e76abcf9c4058f2d036c4bf1cf"
   end
 
   resource "xorg-macros" do
-- 
2.25.0

